llvm/lib/SYCL/XOCCIRDowngrader/XOCCIRDowngrader.cpp

```diff
@@ -62,20 +62,20 @@ struct XOCCIRDowngrader : public ModulePass {
   /// Clang/LLVM 9.
   ///
   /// This LLVM patch primarily added byval <Type> syntax in the IR. And a new
-  /// Attrbiute type, which is not to be confused with anything related to the
+  /// attrbiute type, which is not to be confused with anything related to the
```

Comment:
I am unsure it is a great progress... ;-)

Reply:
didn't realize it was misspelled as well, sorry. Which is irritating as I had my spell checker on...

Reply:
should be fixed now